smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -271,9 +271,17 @@ def initialize() {
 	updateDevices()
 	// Check for new devices and remove old ones every 3 hours
 	runEvery5Minutes('updateDevices')
```

Comment:
If `updated()` is called multiple times, will this schedule multiple `updateDevices`?